Jenkinsfile.deploy

```diff
@@ -99,7 +99,7 @@ pipeline {
                 script {
                     echo 'Setup Maven release config'
                     configFileProvider([configFile(fileId: maven.getSubmarineSettingsXmlId(), targetLocation: 'maven-settings.xml', variable: 'MAVEN_SETTINGS_FILE')]) {
-                        sh "echo '-B -s ${MAVEN_SETTINGS_FILE}' | tee optaplanner/.mvn/maven.config"
+                        sh "echo '\n-B -s ${MAVEN_SETTINGS_FILE}' | tee -a optaplanner/.mvn/maven.config"
```

Comment:
should add also here the `sh "echo '\n-Denforcer.skip=true' | tee -a .mvn/maven.config"` command or ?